### ISSUE TITLE
Travis tests fail as new version of pandas was released and there is no upper restriction for pandas in setup.py

### ISSUE THREAD

**Issue by SabineHaas** [wind-python/windpowerlib#38]
All tests fail as there was a new pandas release on Jan 25 (0.24.0), which causes the following error in the examples:
```
weather_df.index = pd.to_datetime(weather_df.index).tz_localize(
            'UTC').tz_convert('Europe/Berlin')
E       AttributeError: 'Index' object has no attribute 'tz_localize'
```

I would restrict pandas to `pandas >= 0.19.1, < 0.24` and check later if we can adapt the code to work with all versions. Do you agree?

How should we handle this for v0.1.0?

**Comment by uvchik** [comment]
I think we should release v0.1.1 with...

... pandas restricted to < 0.24
... problems solved and pandas restricted to < 0.25 to avoid such problems in the future.

**Comment by uvchik** [comment]
I think it would be best to fix it:

https://pandas-docs.github.io/pandas-docs-travis/whatsnew/v0.24.0.html#parsing-datetime-strings-with-timezone-offsets

**Comment by uvchik** [comment]
Okay I fixed it in #39 and limited the version of pandas to < 0.25 which is the next release with possible API changes.
